src/sentry/api/urls.py

```diff
@@ -567,6 +573,16 @@
                     OrganizationAlertRuleTriggerIndexEndpoint.as_view(),
                     name="sentry-api-0-organization-alert-rules-triggers",
                 ),
+                url(
+                    r"^(?P<organization_slug>[^\/]+)/alert-rules/(?P<alert_rule_id>[^\/]+)/triggers/(?P<alert_rule_trigger_id>[^\/]+)/actions/(?P<alert_rule_trigger_action_id>[^\/]+)/$",
+                    OrganizationAlertRuleTriggerActionDetailsEndpoint.as_view(),
+                    name="sentry-api-0-organization-alert-rule-trigger-action-details",
+                ),
+                url(
+                    r"^(?P<organization_slug>[^\/]+)/alert-rules/(?P<alert_rule_id>[^\/]+)/triggers/(?P<alert_rule_trigger_id>[^\/]+)/actions$",
+                    OrganizationAlertRuleTriggerActionIndexEndpoint.as_view(),
+                    name="sentry-api-0-organization-alert-rules-trigger-actions",
```

Comment:
Do trigger actions need their own endpoints? Could they exist entirely as embedded resources in the trigger? Having a separate HTTP resource makes the server code simpler, but pushes orchestration of trigger rule and action changes onto the client where it could be more error prone and expensive due to network overhead.

Reply:
I discussed this with @billyvg and he preferred individual endpoints at the time